### DISCUSSION TITLE
新版本干净阅读win10不能用吗？

### DISCUSSION THREAD

**Discussion by waitmeyzw** [Clean-Reader/CleanReader.Desktop#4] [Ideas]
![1](https://user-images.githubusercontent.com/67434432/162602177-dfc9b0c9-86eb-43fd-bde0-e3dd669f7356.png)

**Comment by Richasy** [comment]
是的，UWP的话还可以降级，目前Windows APP SDK只有在win11上才能保证不出大问题
